app/Main.hs

```diff
@@ -20,6 +23,9 @@ defaultSettings = Settings { bindAddress = "0.0.0.0"
                            , bufferSize = 2^11
                            , authentication = ""
                            , realm = ""
+                           , https = False
```

Comment:
I think `https` should have a type with 2 constructors: `HTTP` and `HTTPS FilePath FilePath`